### PR TITLE
docs: add licenses and copyrights

### PR DESCRIPTION
**Pull request by lucab**

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: envsubst
+Source: https://github.com/lucab/envsubst-rs
+
+Files: *
+Copyright: 2017-2019, Project contributors
```
Comment (rfairley) [inline COPYRIGHT]: Minor: should this be `2019` instead of `2017-2019`?

Comment (lucab) [reply COPYRIGHT]: Yup, updated.

**Review by rfairley — APPROVED** [review]
LGTM

**Pull request merged**